sdk/lib/convert/html_escape.dart

```diff
@@ -38,6 +38,23 @@ const HtmlEscape htmlEscape = HtmlEscape();
 ///
 /// Custom escape modes can be created using the [HtmlEscapeMode.HtmlEscapeMode]
 /// constructor.
+///
+/// Example:
+/// ```dart
+/// const sample = 'Text & subject, <, >, "quoted", \'single-quoted\', /';
+///
+/// const htmlEscapeMode = HtmlEscapeMode(
+///     name: 'custom',
+///     escapeLtGt: true,
+///     escapeQuot: false,
+///     escapeApos: false,
+///     escapeSlash: false);
```

Comment:
I think it's worth having trailing commas for readability:

```suggestion
///     escapeSlash: false,
/// );
```